includes/Media/Video/Trimming.php

```diff
@@ -74,18 +74,22 @@ public function register_meta() {
 				'show_in_rest'   => [
 					'schema' => [
 						'properties' => [
-							'original' => [
+							'original'  => [
 								'description' => __( 'Original attachment id', 'web-stories' ),
 								'type'        => 'integer',
 							],
-							'start'    => [
+							'start'     => [
 								'description' => __( 'Start time.', 'web-stories' ),
 								'type'        => 'string',
 							],
-							'end'      => [
+							'end'       => [
 								'description' => __( 'End time.', 'web-stories' ),
 								'type'        => 'string',
 							],
+							'elementId' => [
```

Comment:
Was getting a REST error --- will see if this can be removed

Reply:
This definitely needs to be removed.

You're getting the warnings because you are accidentally passing `elementId` to the underlying REST API call.

This happens because you added `elementId` to `trimData` in `trimExistingVideo()`. That needs to be removed.

Reply:
Given most of the code in the PR has been updated to use updateElementById vs updating by Resource ID we might not need to hold the elementId in the trim data at all.

Reply:
Why does this need to saved at all. Couldn't we just delete it before submitted to api?

```
delete trimData.elementId;
```

Reply:
It doesn't need to be saved, so yes it obviously needs to be removed at some point.

I think we are all on the same page that this is just transient data.

Reply:
> Why does this need to saved at all. Couldn't we just delete it before submitted to api?

yes for sure 

> I think we are all on the same page that this is just transient data.

yes

Reply:
You can delete it here

https://github.com/GoogleForCreators/web-stories-wp/blob/27c547c73de98ae361f340cc025122ece0441b9f/packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js#L247